modules/data.atmosphere/R/met.process.R

```diff
@@ -459,13 +459,12 @@ browndog.met <- function(browndog, source, site, start_date, end_date, model, di
                     "<model>", model, "</model>", 
                     "</input>")
   
-  userpass <- paste(browndog$username, browndog$password, sep = ":")
-  curloptions <- list(userpwd = userpass, httpauth = 1L, followlocation = TRUE)
-  result <- RCurl::postForm(paste0(browndog$url, formatname, "/"),
-                     fileData = RCurl::fileUpload("pecan.xml", xmldata, "text/xml"), .opts = curloptions)
-  url <- gsub(".*<a.*>(.*)</a>.*", "\\1", result)
-  PEcAn.logger::logger.info("browndog download url :", url)
-  downloadedfile <- PEcAn.utils::download.url(url, outputfile, 600, curloptions)
+  pecanxml <- tempfile("pecan.xml")
+  write(xmldata, pecanxml)
+  result.url <- BrownDog::convert_file(browndog$url, pecanxml, formatname, "./", browndog$token, download=FALSE)
```

Comment:
what is the ./ represent, if this is a location on disk please use tempfolder just in case the working directory is read-only.

Reply:
no, the file doesn't download so we don't need a folder to hold it. 
it just a placeholder.